config/deploy.rb

```diff
@@ -0,0 +1,14 @@
+lock '3.2.1'
+
+#set :deploy_to, "~/coursemology_deploy"
+set :application, 'coursemology'
+
+set :repo_url, 'git@github.com:coursemology/coursemology.org.git'
+set :branch, 'development'
+set :deploy_via, :remote_cache
+
+set :linked_dirs, ['log', 'tmp/cache', 'tmp/sockets', 'tmp/pids']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.